packages/serum/src/token-mints.json

```diff
@@ -170,5 +170,13 @@
   {
     "address": "SLRSSpSLUTP7okbCUBYStWCo1vUgyt775faPqz8HUMr",
     "name": "SLRS"
+  },
+    {
```

Comment:
indentation is off

Reply:
thanks for your patience, as you can see I'm new to this 😅